src/test/ui/parser/suggest-removing-semicolon-after-impl-trait-items.stderr

```diff
@@ -0,0 +1,18 @@
+error: non-item in item list
+  --> $DIR/suggest-removing-semicolon-after-impl-trait-items.rs:4:16
+   |
+LL | trait Foo {
+   |           - item list starts here
+LL |     fn bar() {};
+   |                ^ non-item starts here
+LL | }
+   | - item list ends here
+   |
+help: consider removing this semicolon
```

Comment:
Can we put this suggestion to the `;` position?
Just found out rust-analyzer does a good job：

![image](https://user-images.githubusercontent.com/230646/184330367-5a1fe8a5-fae4-4956-8144-6a052ea6c69b.png)

Reply:
I also think that putting the suggested help with the arrow that says "non-item starts here" (next line perhaps) is more ideal.